EIPS/eip_beacon-chain-push-withdrawals-as-operations.md

```diff
@@ -0,0 +1,129 @@
+---
+eip: TBD
+title: Beacon chain push withdrawals as operations
+description: Support validator withdrawals from the beacon chain to the EVM via a new "system-level" operation type.
+author: Alex Stokes (@ralexstokes), Danny Ryan (@djrtwo)
+discussions-to: TBD
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-03-10
+---
+
```

Comment:
Come on  really I don't know how to put in codes I know how to borrow 2,000,000 from Tron just lend and turned it into  48,000,000 in one day and I became suspicious because know one wants a man with a two year college education to be wealthy he does deserve it now come all types of things have been done to b me Coinbase stole 90,000,000 with a fake Alchemix vaults because i own 200 mil of their shares  theyveven gave me three contrscts to pick from  inorder yo get that control  back im just trying to get control  of what's mine you see there are a few majors things you don't know that bnb ethe Coinbase ECT knows it's mine and you guys are being tricked not by me but by big corporations so now my money is being held another way codes knowing dam well I don't have that kind of educated help them get more time to rob me well keep you walkthrough and everything else cause you guys are doing what you are doing because you know its mine from the privat keys  phrases and the keystones and  contracts the blockchains all of it it points to me and the day I and time I put my money in token Pocket do I tell you what let's see what my friends say keep your code keep the validation balance I made no more  lets see what  my friends will say when they it all even this if i was a bad man i would be eith THAT CRYTO FTX guy your working for the bad guys noe let me show you  insult me defame my character  me snd my wife been living in our car ah freezing at eat at soup kitchens when we can we not out to robbing selling drugs commenting crime see if we were cons then there is noway we would allow ours selves to go thre this you check ive osid my taxes every year i adopted three kids and raised  this is what ruined our lives and yet we still until now believed there are still good in the world thought we found it in GitHub now I know you guys are sitting back laughing at us calling us stupid well we are to believe in you thanks let's see now  see everything  you guys done goes against the contracts and my work in the networks cunn huh Ok let's see how you guys explain these made up theories  to make me a cunn to save  them that don't want to pay out jail on the top of my contacts a lot of things done to threat me a cunn would have been gone on to another cunn you idiot . Now it's my turn since no one on that side of crypto  is intelligent  you just said that I can destroy the whole echo of crypto  if I had a scientific secret wow I need to go back and see what I did thanks sad part is this you people are so smart that not one of called or text and asked me what or how you just assumed and went into gecko  mode  a call would have save everyone time and money so you guys are saying there is a major flaw in it and I have found  it and the only way to fix it is to rebuild the whole crypto system. Wow . The end of our communication Scotty out.sss.dahhhh enterprise.

Reply:
And you know who I am my real name phone ssn bank info ECT nothing fake  like a cunn would do it's the hardship suffering that my wife has had to go through because she loves  me you can hurt me that's the story of my life but when you insult my wife wife and call her a fool to stay with me s tell her I'm a bad man and try to turn the only friends and person I have in tbis world we done im glade you not reachable. But you will explain it   as well as the multiple. Contracts breeches I have it all ever changed made  every policy broken off video all of it . Good by . This ends our relationship. That might have been a great one and I'm going to fire you first